### PR TITLE
DEV-3882-docs: Write readme for byebyefirst subject

### PR DESCRIPTION
**Pull request by eslopfer**
Write readme for byebyefirst subject.

### CODE REVIEW

**Review by Hamzaelkhatri — APPROVED** [review]

File: byebyefirst/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+### Usage
+
+Here is a possible program to test your function :
+
+```go
+package main
+
+func main() {
+	fmt.Println(ByeByeFirst([]string{}))
```
Comment (MSilva95) [inline byebyefirst/README.md:22]: Just add this example and its good to go:
	
```fmt.Println(ByeByeFirst([]string{"one arg"}))```

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: byebyefirst/README.md (hunk 1 above)
Comment (jotapero) [reply byebyefirst/README.md]: added

**Review by jotapero — APPROVED** [review]
Looks Good :alien:

**Review by tiagocollot — APPROVED** [review]
LGTM! ⚡

**Review by tiagocollot — CHANGES_REQUESTED** [review]

**Review by tiagocollot — APPROVED** [review]
LGTM ⚡

File: subjects/byebyefirst/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+## byebyefirst
+
+### Instructions
+
+Write a function that takes a slice of `string` and returns a new slice without the first element.
```
Comment (tiagocollot) [inline subjects/byebyefirst/README.md:5]: Should be slice of `strings`---> plural.

**Review by tiagocollot — CHANGES_REQUESTED** [review]

File: subjects/byebyefirst/README.md (hunk 2 above)
Comment (jotapero) [reply subjects/byebyefirst/README.md]: fixed

**Review by tiagocollot — APPROVED** [review]
LGTM! ⚡

**Pull request merged**